playbooks/openshift-node/private/certificates.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Create OpenShift certificates for node hosts
-  hosts: oo_nodes_to_config:!oo_nodes_to_bootstrap
```

Comment:
@sdodson I'm not familiar with this play, but we should probably backport?  It appears there's no way to replace certs on nodes ATM.